tests/run/profile/test.sh

```
@@ -0,0 +1,38 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        tmt="tmt --root data -vv test export"
+
+        rlRun -s "$tmt --profile profiles/test/test.yaml /basic 2> /dev/null | yq -cSr '.[] | .test'"
+        rlAssertEquals "foo?" "$(cat $rlRun_LOG)" "scl enable gcc-toolset-15 dummy-command"
```

Shellcheck (0.10.0)

[warning] rlRun_LOG is referenced but not assigned.

(SC2154)